contentcuration/contentcuration/frontend/channelEdit/components/ResourcePanel.vue

```diff
@@ -3,13 +3,18 @@
   <VLayout row wrap @scroll="$emit('scroll', $event)">
     <VFlex xs12>
       <VLayout v-if="!hideNavigation" row>
-        <VFlex v-if="!loading && node">
-          <div class="mb-1">
-            <!-- Slot for elements like "Back" link -->
-            <slot name="navigation"></slot>
-          </div>
-          <ContentNodeIcon :kind="node.kind" includeText />
-        </VFlex>
+        <div v-if="!loading && node" class="mb-1">
+          <!-- Slot for elements like "Back" link -->
+          <slot name="navigation"></slot>
+          <ContentNodeLearningActivityIcon v-if="isTopic" :isTopic="true" includeText chip />
+          <ContentNodeLearningActivityIcon
+            v-else-if="hasLearningActivities"
```

Comment:
Not blocking, this can stay as is. Just an idea for rows 9-15 if you'd find it helpful. Maybe having another component, `TopicIcon`, separated from `ContentNodeLearningActivityIcon` would help with understanding and using `ContentNodeLearningActivityIcon`.